core/modules/filter/filter.module

```diff
@@ -1410,17 +1410,11 @@ function filter_process_format($element) {
  */
 function filter_get_attached($formats) {
   $attached = array();
-  $attached['library'][] = array('filter', 'filter');
```

Comment:
This library is the *filter* library, removing it is probably not intended, please restore that single line.

Regarding the removal of that non-existent "$added" variable: :+1:

Reply:
Are you sure the line was intended as I've run all kind of tests without this line hitting no issues, no errors?

Reply:
Hm, I'll give it a try in the sandbox, stay tuned. :wink:

Reply:
It still seems to get attached, but... I remember, that we had a related issue in the past when the filter library was missing.

For my peace of mind - please add it back. :smile:

Reply:
I've reverted the changed, though that line does nothing from my perspective. The library you see gets attached elsewhere and has something to do with filter, but editor. Lot's of things got mixed between the two in this function.